test/dispatchers/dispatcherTests.ts

```diff
@@ -205,6 +205,67 @@ describe("Dispatchers", () => {
       target.remove();
     });
 
+    it("onMouseDown()", () => {
+      var targetWidth = 400, targetHeight = 400;
+      var target = generateSVG(targetWidth, targetHeight);
+      // HACKHACK: PhantomJS can't measure SVGs unless they have something in them occupying space
```

Comment:
Any chance we can file an issue on them or something so we can remove the hackhack in the future?